ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/passthrough/PassThroughHandler.java

```diff
@@ -33,12 +35,22 @@
 
   private final HttpClient ethNodeClient;
   private final VertxRequestTransmitter transmitter;
+  private final String httpRequestArgs;
+
+  private final String DEFAULT_URI = "/";
```

Comment:
This class could be simplified if the default behaviour was removed - i.e. enforce all users of this class to specify the value (this is already defaulted on the cmdline).
If the default behaviour is really important - then it may be worth offering 2 constructors, one which takes a configurable httpPath and another which doesn't (but invokes the first with "/".

Reply:
Simplified the class but added an additional flag to support 3 different cases:
1. default case - use the URI
2. infura case - replace the URI with the path
3. extension case - include both the additional path and the URI

Added test cases to validate all 3 cases and changes.